BinariesCheck.py

```diff
@@ -484,6 +486,9 @@ def check_binary(self, pkg):
                 for ec in bin_info.exit_calls:
                     printWarning(pkg, 'shared-lib-calls-exit', fname, ec)
 
+            if bin_info.lto_sections:
+                printError('E', pkg, 'lto-bytecode', fname)
```

Comment:
this doesn't look correct, the proper usage would be:

```
printError(pkg, 'lto-bytecode', fname)
```

Reply:
Fixed.